#!/usr/bin/env python

PACKAGE = "cargo_drone"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_kp", double_t, 0, "A double parameter",    .1, 0,   1)
gen.add("y_kp", double_t, 0, "A double parameter",    .1, 0,   1)
gen.add("z_kp", double_t, 0, "A double parameter",    .1, 0,   1)
gen.add("ar_timeout", double_t, 0, "A double parameter",    5, 0,   10)
gen.add("diameter_thresh", double_t, 0, "A double parameter",    .2, 0,   1)
gen.add("lower_target_alt", double_t, 0, "A double parameter",    2, 0,   4)
gen.add("upper_target_alt", double_t, 0, "A double parameter",    8, 0,   15)
gen.add("takeoff_alt", double_t, 0, "A double parameter",    8, 0,   15)

exit(gen.generate(PACKAGE, "controller", "drone_param"))